scripts/engineering/generate_phase44d_carry_aware_experimental_3x3_scaling_figure.py

```
@@ -0,0 +1,361 @@
+        if runs != 1:
+            raise SystemExit(
+                f"measured_single_run phase44d engineering rows must report timing_runs == 1; got {runs}"
+            )
+        return mode, runs
+    if mode == "measured_median":
+        if not policy.startswith("median_of_") or not policy.endswith(
+            "_runs_from_microsecond_capture"
+        ):
+            raise SystemExit(
+                f"unexpected timing_policy for measured_median phase44d engineering rows: {policy!r}"
+            )
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)